editor/find_in_files.cpp

```diff
@@ -125,6 +125,9 @@ void FindInFiles::start() {
 		return;
 	}
 
+	if (_whole_words) {
+		_pattern = _pattern.strip_edges();
+	}
```

Comment:
Just a minor comment, but this looses information in a race condition.
Considering the API of this class, the following could happen:
```cpp
find_in_files.set_search_text("  Test");
find_in_files.set_whole_words(true);
find_in_files.start();
...
find_in_files.set_whole_words(false);
find_in_files.start();
// This will look for `Test` but without the original spaces, without the caller knowing
```
However it's probably not an issue because this sequence of calls does not happen currently, the text is set again with all other properties in `ScriptEditor::_start_find_in_files`.